chapters/methodology.tex

```diff
@@ -9,8 +9,20 @@ \subsection{Architecture}
 Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
 \subsection{State Machine}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
 
-\subsection{User Interface}
-Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum.
+\begin{figure}
+  \includegraphics[width=\linewidth]{figures/state_diagramm.jpg}
+  \caption{The state diagram.}
+  \label{fig:state_diagramm}
+\end{figure}
+
 
+Figure \ref{fig:state_diagramm} shows the states and transitions of the state machine.
+A state change can be invoked through three different kind of actions: An user action (for example pressing a button), a timeout after a specified timeframe or a significant analyser result (face detection or smile detection).
+
+Note that if a face is detected and users move their faces out of the frame afterwards, the process continues and handles this event indirectly as missing smile.
+Consistently, the face leaving the camera frame after a first smile detection is not dealed with explicitly but leads to low smile results.
+Furthermore, instead of an end knot, the application flow returns to the start screen, represented in the start state.

```

Comment:
- Is `Furthermore` really necessary in this context? I'd start with `Instead of having an end knot, the application...`. Though this decision depends on personal preferences...
- I'd use `node` instead of `knot`